extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { noop } from 'lodash';
```

Comment:
I'm seeing the entirety of `lodash` being included in the built scripts, but can't find a way to test if it shows up in the production build or not. If this _is_ including the entirety of `lodash` in the build instead of just the `noop` function, could we do: `const noop = () => undefined;` instead of including it from `lodash`?

Reply:
You can run `yarn build-production` to get the production build.

Reply:
It's also tricky because we have some magical minification and concatenation done on the fly in wpcom, right? 

For now, since we have only used `lodash` for the noop, I've done a simple local replacement as suggested.

Reply:
@jeherve As an FYI, I was using `NODE_ENV=production yarn build-client`, as I had found from these docs: https://github.com/Automattic/jetpack/blob/master/docs/development-environment.md#production-build

Reply:
Yeah, we'd need to update those docs since they were created before blocks were a thing.

Reply:
I tried rebuilding using `yarn build-production`, and I don't see minified files locally. Also, @marekhrabe, `lodash` is still enqueued even with this change, so I'm guessing lodash is already being added via one of the other dependencies automatically. It could be a huge performance gain if this was able to be addressed in a way where when you do `import { noop } from 'lodash';` it _only imported_ `noop`.

Any ideas on who might know about this implementation?

Reply:
> I don't see minified files locally

You should see them at `_inc/blocks/podcast-player`.

The Build configuration is available here:
https://github.com/Automattic/jetpack/blob/c9e2901cd559ec75322abd21a50d6e00eddbee78/webpack.config.extensions.js

It relies on Calypso Build:
https://github.com/Automattic/wp-calypso/tree/master/packages/calypso-build

I believe it's maintained by Team Calypso.

> It could be a huge performance gain if this was able to be addressed in a way where when you do import { noop } from 'lodash'; it only imported noop.

I would expect this to be the default behaviour too, so if this is not happening that is definitely a bug imo.